camkes_vm_helpers.cmake

```diff
@@ -257,20 +261,22 @@ endfunction(DeclareCAmkESVMRootServer)
 # Parameters:
 #
 # <filename_pref>
-#   The name the caller wishes to use to reference the file in the CPIO archive.
-#   This corresponds with the name set in the 'kernel_image' camkes variable for
-#   a given instance vm.
+#   The name to use for the file in the file server
```

Comment:
Is the part about kernel_image etc no longer true? If not I'd leave that in to make the connection clearer.

Reply:
I found it more confusing that helpful to mention this here. It seems that `kernel_image` is only used for the x86 VMM, the ARM VMM uses either `linux_image_config.linux_name` (old, deprecated) or `vm_image_config.kernel_name`(new) now. The VMMs are diverged a bit, unifying things might be worth a thought...

Reply:
I've added a generic note now.
Also removed mentioning a term "archive" when talking about the file server's content, as using CPIO archive is just another implementation detail.